packages/in_app_purchase/in_app_purchase_storekit/example/shared/RunnerTests/InAppPurchasePluginTests.m

```diff
@@ -419,6 +419,71 @@ - (void)testGetPendingTransactions {
   XCTAssertEqualObjects(resultArray, @[ transactionMap ]);
 }
 
+- (void)testPaymentQueueStorefront {
+  if (@available(iOS 13, macOS 10.15, *)) {
+    {
```

Comment:
Nit: These should be two separate test methods, rather than sub-blocks of the same test. Unit tests should each test as few things as is feasible.